protocol/x/clob/keeper/untriggered_conditional_orders.go

```diff
@@ -323,21 +323,19 @@ func (k Keeper) MaybeTriggerConditionalOrders(ctx sdk.Context) (triggeredConditi
 	return triggeredConditionalOrderIds
 }
 
-// CountUntriggeredSubaccountOrders will count all untriggered orders for a given subaccount that match the provided
-// filter.
-func (k Keeper) CountUntriggeredSubaccountOrders(ctx sdk.Context,
+// CountUntriggeredSubaccountStatefulOrders will count all untriggered stateful orders for a given subaccount.
```

Comment:
Suggestion, since conditional orders can only be stateful:
```suggestion
// CountUntriggeredSubaccountConditionalOrders will count all untriggered conditional orders for a given subaccount.
```